src/core/external/collect/collect-element.ts

```diff
@@ -101,9 +109,13 @@ class CollectElement extends SkyflowElement {
     this.elementType = this.#isSingleElementAPI
       ? this.#elements[0].elementType
       : 'group';
-
+    initalizeMetricObject(metaData, elementId);
+    if (!this.#isSingleElementAPI) {
+      updateMetricObjectValue(this.#elementId, 'element_type', this.elementType);
```

Comment:
better to move values like 'element_type' , `mount_end_time` etc all hardcoded values to enums 
so that it will be always one place to change.

Reply:
Sure Will change that